docs/source/reference/terminator.rst

```diff
@@ -18,4 +18,6 @@ The :mod:`~optuna.terminator` module implements a mechanism for automatically te
    optuna.terminator.CrossValidationErrorEvaluator
    optuna.terminator.StaticErrorEvaluator
    optuna.terminator.TerminatorCallback
-   optuna.terminator.report_cross_validation_scores
\ No newline at end of file
+   optuna.terminator.report_cross_validation_scores
+
+For an example of using the this module, please refer to `this example <https://github.com/optuna/optuna-examples/tree/main/terminator>`_.
```

Comment:
My bad, sorry.
```suggestion
For an example of using this module, please refer to `this example <https://github.com/optuna/optuna-examples/tree/main/terminator>`_.
```

Reply:
> My bad, sorry.

No sir , it was completely my fault, I didn't notice that properly, sorry for inconvenience caused.
Sir, I can I contact you via e-mail? i want to discuss something.

Reply:
Thank you for updating the PR! 

> E-mail

I'm glad to hear that but I'm afraid I cannot. For optuna-related things, I'm not the right person because I'm not a member of the core developer team. But of course, I'm happy to support new contributors on Github like this PR's review :)

Reply:
> I'm glad to hear that but I'm afraid I cannot. For optuna-related things, I'm not the right person because I'm not a member of the core developer team. But of course, I'm happy to support new contributors on Github like this PR's review :)

No problem Sir, I will try discuss with someone, from core team member. Thanks for your acknowledgement.